website/client/components/challenges/challengeItem.vue

```diff
@@ -1,6 +1,7 @@
 <template lang="pug">
 .card
   .row
+    span(v-if='canViewFlags') {{ $t('hasBeenFlagged') }}
```

Comment:
I'm sorry, I should have noticed this before. This needs to say either "Challenge flagged once, not hidden" or "Challenge hidden" (see https://github.com/HabitRPG/habitica/issues/9725 for details). The basic "Flagged" message was specified in https://github.com/HabitRPG/habitica/issues/5557 but that was for the case where hiding wasn't implemented. Mods need to know if a challenge has been flagged enough to hide it.

By the way, if you'd prefer to not make it translatable text, that's okay since it will be seen by only moderators - I really should have thought of that when I got you to change it from "flagged" to "hasBeenFlagged" :(

That text should also appear on the challenge's own page (again, only when a moderator views the challenge).

Reply:
"Challenge flagged once, not hidden" and "Challenge hidden" strings could go into website/server/libs/apiMessages.js since they don't need to be translated.